jobs/release-microk8s-capi.yaml

```
@@ -0,0 +1,60 @@
+# Builds and releases microk8s capi providers
+
+- job:
+    name: 'release-microk8s-capi'
+    description: |
+      Build and release MicroK8s capi.
+    project-type: freestyle
+    node: runner-amd64
+    wrappers:
+      - ansicolor
+      - timestamps
+      - ci-creds
+      - workspace-cleanup
```

the `default-job-wrapper` handles these other three wrappers

```suggestion
    wrappers:
      - default-job-wrapper
      - ci-creds
```